### PR TITLE
Docs/readme: minor changes & bump Cargo version

### PR DESCRIPTION
**Pull request by utkarshgupta137**

### CODE REVIEW

**Review by lunacookies — APPROVED** [review]

**Comment by lunacookies** [comment]
Overall looks like a nice improvement! Feel free to merge whenever you like :)

File: README.md [hunk 1]
```diff
@@ -27,10 +31,12 @@ Etcetera has 2 modes of operation: `BaseStrategy` & `AppStrategy`:
...
 
+###
 Etcetera also provides convenience functions for selecting the appropriate strategy on each platform:
 - `base_strategy::choose_base_strategy` & `app_strategy::choose_app_strategy`: Uses `Windows` on Windows & `XDG` everywhere else.
   This is used by most CLI tools & some GUI tools on each platform.
 - `base_strategy::choose_native_strategy` & `app_strategy::choose_native_strategy`: Uses `Windows` on Windows, `Apple` on macOS/iOS, & `XDG` everywhere else.
   This is used by most GUI applications on each platform.
 
-See the documentation for examples.
+##
```
[the remark this answers is not in the document]
Comment (utkarshgupta137) [reply README.md]: The 1st one shouldn't have been empty, fixed.
The 2nd is to add a small divider.

**Comment by utkarshgupta137** [comment]
> Overall looks like a nice improvement! Feel free to merge whenever you like :)

I'm a crates.io noob. I've done the authentication. Should I just do `cargo publish`?

**Comment by utkarshgupta137** [comment]
I've published v0.6.0 :)

**Pull request merged**